### ISSUE TITLE
I can't find any documentation

### ISSUE THREAD

**Issue by nikkilocke** [tumtumtum/Shaolinq#45]
Despite numerous Google searches, I can't seem to find any documentation.

So I have downloaded and installed Shaolinq, and am ready to get started writing code. But I have no idea what to write in order to create classes that ShaoLinq will understand, how to get ShaoLinq to create a database to match those classes, how to create, update, delete and retrieve objects, or anything.

Surely there must be some instructions somewhere, but I can't find them!

**Comment by hahn-kev** [comment]
Same here some documentation would be great

**Comment by InquisitionX** [comment]
I haven't found any Documentation either, but there is a pretty good array of Tests in the code base. On the github Shaolinq page, you'll see the Download Zip button on the right of the page.
 
Take a look at the test app.config, and I guess walking through some of the code in the test project will explain how you should be using the library in your code. This is where I started from:
public void Test_LoadCustomConfig()
and to be sure for my case:
public class PostgresDotConnectSqlDatabaseContext : SqlDatabaseContext
CreatePostgresDotConnectConfiguration

Hope that gets you started.

**Comment by nikkilocke** [comment]
Thanks, I tried that, and gave up because I wanted to do something different, and couldn't work out how.

**Comment by tumtumtum** [comment]
Good call. With implicit join support, include support and async support mostly finished and ready for use I will be focussing on documentation

**Comment by yograterol** [comment]
Yes, the wiki or another place would be better for all documentation, I want to use Shaolinq with PostgreSQL.
